tests/queries/0_stateless/02911_add_index_and_materialize_index.sql

```diff
@@ -15,4 +15,9 @@ ALTER TABLE index_test
     MATERIALIZE INDEX i_x,
     MATERIALIZE INDEX i_y;
 
+ALTER TABLE index_test
```

Comment:
Maybe we should test that indices are actually materialized? For example we can add SELECT query which will read from table with WHERE expression which can get benefits from indices and we will test that we read less rows than full scan. Here is an example where we check how many rows were read https://github.com/ClickHouse/ClickHouse/blob/f2ffd727f002702ab29dff7c2d18ceaaf8e09e6e/tests/queries/0_stateless/02906_force_optimize_projection_name.sql#L27-L33.

Reply:
thanks. updated with the case.